resources/views/external-link.blade.php

```diff
@@ -1,26 +1,25 @@
 @props([
     'url',
     'text',
-    'class'     => 'link font-semibold',
-    'inline'    => false,
-    'allowWrap' => false,
+    'class'     => 'link font-semibold inline',
+    'breakAll'  => false,
     'small'     => false,
     'noIcon'    => false,
 ])
 
 <a
     href="{{ $url }}"
-    class="{{ $class }} {{ $inline ? 'inline space-x-1' : 'flex items-center space-x-2' }} {{ $allowWrap ? '' : 'whitespace-nowrap' }}"
+    class="{{ $class }} {{ $breakAll ? 'break-all' : 'break-words' }} "
```

Comment:
would make it part of the class property instead, so it has `link font-semibold inline break-words`. That way if you really need `break-all` you can pass it as class anyway without needing a special property for it